kuma/users/tests/test_views.py

```diff
@@ -1132,75 +1131,44 @@ def test_email_addresses(self):
         profile_data = self.github_profile_data.copy()
         profile_data["email"] = public_email
         email_data = [
-            {"email": private_email, "verified": True, "primary": True},
+            # It might be unrealistic but let's make sure the primary email
+            # is NOT first in the list. Just to prove that pick that email not
+            # on it coming first but that's the primary verified one.
             {"email": unverified_email, "verified": False, "primary": False},
+            {"email": private_email, "verified": True, "primary": True},
```

Comment:
As of this change, I **think** this automatically answers https://github.com/mdn/kuma/issues/6541
What do you think @escattone ?